### PR TITLE
draft for MS8 workshop + feature picture as thumbnail

### PR DESCRIPTION
**Pull request by SanjeAntona**
draft for article/promotion of the MS8 workshop by WP3
small restructuring for better readability.

reused and adapted WP3 banner for thumbnail in the news overview page.

please check and let me know if anything is amiss.

### CODE REVIEW

File: content/news/2024-12-10-ms8-workshop/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+[Work package 3](everse.software/workpackages/03_tools_and_services/) - _Tools and Services_ is organising an interactive workshop to examine which tools and services researchers use to improve the quality of research software, and how they use them!
+
+**<font color="40A3DC">Save the date: Monday 10th February 2025, 10:00-16:00 CET.** </font> \
+Please register your interest [here](https://forms.office.com/Pages/ResponsePage.aspx?id=sAafLmkWiUWHiRCgaTTcYY8mG3jqdkZDlfFoAQVJfklUNVNQMEVQUDQ4SUg1MVpXRzRIMDZONDYwQi4u). 
+
+
+**Is this workshop for you? Join us if you:**
+* Work with/develop research software.
+* Work with or want to develop software of high quality (considering FAIRness, sustainability, portability, testing, security, etc.).
```
Comment (vuillaut) [inline content/news/2024-12-10-ms8-workshop/index.md:20]: these two lines read redundant 
I'd rephrase:
- Work with or develop research software
- Want to improve your software quality (considering FAIRness, sustainability, portability, testing, security, etc.).

**Review by vuillaut — COMMENTED** [review]
Hi
Thanks for the note.
I have a small comment. You may merge when edited.

File: content/news/2024-12-10-ms8-workshop/index.md (hunk 1 above)
Comment (graeme-a-stewart) [reply content/news/2024-12-10-ms8-workshop/index.md]: Yes I agree - I tried to make it clearer between working with software and striving to improve it, along the lines of what you suggested

**Review by graeme-a-stewart — APPROVED** [review]
